pyvista/plotting/colors.py

```diff
@@ -348,10 +377,473 @@
 PARAVIEW_BACKGROUND = [82 / 255.0, 87 / 255.0, 110 / 255.0]
 
 
-def hex_to_rgb(h):
+class Color:
+    """Helper class to convert between different color representations used in the pyvista library.
+
+    Many pyvista methods accept :attr:`color_like` parameters. This helper class
```

Comment:
Not sure myself...

Reply:
Actually, since `color_like` has full rendered documentation, we should be able to mark it up with an internal link. At this point probably something like `:ref:`pyvista.color_like`? I always need 3-4 tries to get this right.